drivers/native-driver/src/nativeMain/kotlin/com/squareup/sqldelight/drivers/native/NativeSqlDatabase.kt

```diff
@@ -105,19 +104,40 @@ sealed class ConnectionWrapper : SqlDriver {
  * when execute or executeQuery is called. This avoids race conditions with bind calls.
  */
 class NativeSqliteDriver(
-  private val databaseManager: DatabaseManager
-) : ConnectionWrapper(),
-  SqlDriver {
+  private val databaseManager: DatabaseManager,
+  maxReaderConnections: Int = 1,
+  maxTransactionConnections: Int = 1
+) : ConnectionWrapper(), SqlDriver {
+  internal val _maxTransactionConnections: Int = when {
```

Comment:
dont need the underscore here

Reply:
Moving that completely. We only need it for Pool constructor.